src/test/unit/inboundApi/handlers.test.js

```diff
@@ -17,24 +18,28 @@ const handlers = require('../../../InboundServer/handlers');
 const Model = require('@internal/model').InboundTransfersModel;
 const mockArguments = require('./data/mockArguments');
 const mockTransactionRequestData = require('./data/mockTransactionRequest');
+const mockAuthorizationArguments = require('../lib/model/data/mockAuthorizationArguments.json');
 const mockLogger = require('../mockLogger');
 const AuthorizationsModel = require('@internal/model').OutboundAuthorizationsModel;
+const ThirdpartyTrxnModelIn = require('@internal/model').InboundThirdpartyTransactionModel;
+const ThirdpartyTrxnModelOut = require('@internal/model').OutboundThirdpartyTransactionModel;
 
 
 describe('Inbound API handlers:', () => {
     let mockArgs;
     let mockTransactionRequest;
+    let mockAuthReqArgs;
 
     beforeEach(() => {
         mockArgs = JSON.parse(JSON.stringify(mockArguments));
         mockTransactionRequest = JSON.parse(JSON.stringify(mockTransactionRequestData));
```

Comment:
let define `function deepClone(obj) { return JSON.parse(JSON.stringify(obj)); }` and use it

Reply:
Ok